modules/dxp/apps/portal-workflow/portal-workflow-metrics-lang/src/main/resources/content/Language.properties

```diff
@@ -107,6 +107,7 @@ on-time=On Time
 once-there-are-active-processes-metrics-will-appear-here=Once there are active processes, metrics will appear here.
 one-or-more-slas-are-being-updated=One or more SLAs are being updated.
 open=Open
+open-page-in-new-tab=Open page in new tab
```

Comment:
Hey @naraandrade,

Add `.` at the end of the sentence.

thanks!

Reply:
Should it be `Open page in a new tab.`?

Reply:
According to the mockup reference on the ticket, it's `Open page in new tab`

Reply:
Okay, but it won't pass.

Please see: https://github.com/brianchandotcom/liferay-portal/pull/58629#issuecomment-390004381

Reply:
Oh, then I think it would be a title. `Open Page In New Tab`. What do you think, @inacionery ?

Reply:
Can be, but it will be `Open Page in a New Tab`. Use [titlecaseconverter](https://titlecaseconverter.com/) APA

Reply:
Ok, thank you :)